SeleniumCookies/wrapper.py

```diff
@@ -356,27 +364,30 @@ def chrome(cookie_file=None, domain_name=""):
     """Returns a cookiejar of the cookies used by Chrome. Optionally pass in a
     domain name to only load cookies from the specified domain
     """
+    logging.info('Loading cookie from Chrome')
     return Chrome(cookie_file, domain_name).load()
 
 
 def firefox(cookie_file=None, domain_name=""):
     """Returns a cookiejar of the cookies and sessions used by Firefox. Optionally
     pass in a domain name to only load cookies from the specified domain
     """
+    logging.info('Loading cookie from firefox')

```

Comment:
Not needed in production. Can be used in `DEV`/`DEBUG` mode. In fact all `logging.info` can be removed in `PROD` mode. `logging.warn` and `logging.error` fields are necessary in production.

```python
environment = os.environ.get("ENV")
if environment == "PROD":
  # Use warn and errors
else:
  # Use debug and info as well
```

For implementing this, you can create wrappers around the following functions:

* logging.info
* logging.warn
* logging.error
* logging.debug

So for example if a function `loggerWrapper.info("Hello World")` is called then internally it should only invoke the `logging.info` construct if the environment is `DEV`/`DEBUG`.

Reply:
I have set logging level for this, it can be set to upper level to supress everything below that